Automattic/wp-calypso
# Help Center: clear Wapuu chat after contacting support

Fixes https://github.com/Automattic/hal-tasks/issues/71

## Proposed Changes

Clear Odie Chat ID after Zendesk is opened

## Why are these changes being made?

Currently the Wapuu chat session snowballs and makes it difficult to parse when the user contacts and HE. This will clear the session after they reach Zendesk. This will also prevent the user from getting the option to Contact Support again without going through Wapuu.

## Testing Instructions

1. Go to Wapuu and asking to speak with a human.
2. Click the contact support button.
3. Open Wapuu again and see if the chat is clear.
4. Make sure the chat session doesn't reappear.
5. I would maybe check Zendesk as well to make sure this doesn't cause any issues with what is passed through,\.
